content/en/docs/tasks/administer-cluster/reserve-compute-resources.md

```diff
@@ -146,6 +146,28 @@ control group (`system.slice` on systemd machines for example).
 Note that Kubelet **does not** create `--system-reserved-cgroup` if it doesn't
 exist. Kubelet will fail if an invalid cgroup is specified.
 
+### Explictly Reserved CPU List
+{{< feature-state for_k8s_version="v1.17" state="stable" >}}
+
+- **Kubelet Flag**: `--reserved-cpus=`
+
+`reserved-cpus` is meant to define an explict CPU set for OS system daemons and
+kubernetes system daemons. This option is added in 1.17 release. `reserved-cpus`
+is for systems that do not intent to define seperate top level cgroups for
+OS system daemons and kubernetes system daemons with regard to cpuset resource.
+If the Kubelet **does not** have `--system-reserved-cgroup` and `--kube-reserved-cgroup`,
+the explicit cpuset provided by `reserved-cpus` will take precedence over the CPUs
+defined by `--kube-reserved` and `--system-reserved` options.
+
```

Comment:
What goes after the `=` sign? I'd guess that I could specify, eg: `--reserved-cpus=8-15` but I'm not sure if I've guessed correctly.

Reply:
That's right. Updated.